Makefile.js

```diff
@@ -36,14 +36,19 @@ var NODE = 'node ',	// intentional extra space
 
 	// Since our npm package name is actually 't3js'
 	DIST_NAME = 't3',
-	DIST_TESTING_BUNDLE_NAME = 't3-testing',
+	DIST_JQUERY_NAME = DIST_NAME + '-jquery',
+	DIST_NATIVE_NAME = DIST_NAME + '-native',
```

Comment:
Actually before you guys merge, realize that adding -foobar will change the bundle name, and will break AMD requiring. Perhaps not use the -native bundle name. So that you have t3.js which is native and t3-jquery.js which is non-native. So at least the native one can be required in.